src/styles/fullcalendar.css

```diff
@@ -0,0 +1,21 @@
+.fc-today-button {
+  display: none !important;
+}
+
+.fc-prev-button {
+  background-color: #4a5568 !important;
+  border-color: #718096 !important;
```

Comment:
Codacy found an issue: [Use of !important](https://app.codacy.com/gh/treetoplearning/heartwood/pullRequest?prid=5837031)